### ISSUE TITLE
Workshop Section on OSP Entry-Page

### ISSUE THREAD

**Issue by StephanSchaller** [Open-Systems-Pharmacology/Forum#268]
There are already two workshops planned for the OSP Suite this year:

https://github.com/Open-Systems-Pharmacology/2-Day-WASHINGTON-Workshop-2019
https://github.com/Open-Systems-Pharmacology/BERLIN-2019-WORKSHOP

We are planning on another 5 until end of year.
To get a better overview for users, maybe we can add a Workshops Section on 
http://www.open-systems-pharmacology.org/

**Comment by Yuri05** [comment]
Good idea. We could either create a new section or split the current `Tutorials` section in 2 parts: _Tutorials_ and _Workshops_ (like in `Publications&Models`). The section would be renamed in `Tutorials&Workshops` (or `Education`?)
